### DISCUSSION TITLE
希望可以添加播放通道选择功能

### DISCUSSION THREAD

**Discussion by unmmx** [SPlayer-Dev/SPlayer#199] [想法 Ideas]
希望可以添加播放通道选择功能,类似于网易云输出设备一样的功能!
![Uploading 微信截图_20240702052857.png…]()

**Comment by unmmx** [comment]
![微信截图_20240702052857](https://github.com/imsyy/SPlayer/assets/60911006/da7c3630-337d-4cbb-adde-1c8d4a9db360)

**Comment by imsyy** [comment]
#198 会在新版本加入该功能
